evalml/tests/automl_tests/test_automl_search_regression.py

```diff
@@ -82,7 +82,7 @@ def add_result_callback(results, trained_pipeline, automl_obj, counts=counts):
                           add_result_callback=add_result_callback, n_jobs=1)
     automl.search()
 
-    assert counts["start_iteration_callback"] == max_iterations
+    assert counts["start_iteration_callback"] == len(get_estimators("regression")) + 1
```

Comment:
Interesting, why did this have to change? Were we not calling this for the baseline pipeline before? That seems like a positive change to me.

Reply:
Correct!